pygnmi/client.py

```diff
@@ -471,12 +471,19 @@ def get(
             raise gNMIException("Conversion of gNMI paths to the Protobuf format failed", e)
 
         try:
-            gnmi_message_request = GetRequest(
-                prefix=protobuf_prefix,
-                path=protobuf_paths,
-                type=pb_datatype,
-                encoding=pb_encoding,
-            )
+            if prefix is None and target is None:
```

Comment:
why check for target here ? previous checks are not taking that into account. 

Will this introduce any change in the behavior, if yes please document what it would be and check if it is acceptable.

Reply:
Previously, the prefix block was present in every request. Now, this change is to allow prefix block to be optional. The 'target' field is present only in the prefix block. Hence, we check for target field here.